playbooks/provisioning/openstack/sample-inventory/group_vars/OSEv3.yml

```diff
@@ -5,7 +5,7 @@ openshift_master_default_subdomain: "apps.{{ env_id }}.{{ public_dns_domain }}"
 
 # NOTE(shadower): do not remove this line, otherwise the default node labels
 # won't be set up.
-openshift_node_labels: "{{ openstack.metadata.node_labels }}"
+openshift_node_labels: "{{ metadata.node_labels }}"
```

Comment:
There is already `nodes` group that takes the app labels, nothing more to add I suppose